mongodbatlas/data_source_mongodbatlas_alert_configurations.go

```diff
@@ -0,0 +1,146 @@
+package mongodbatlas
```

Comment:
can you also build a _test.go file to go along with this?

Reply:
Yep! Test build which ensures:
- The data_source pulls in all alerts (within the page limit)
- data_source applies twice with no changes (had an issue here originally with formatting of the output changing between applies)